integration-tests/src/test/java/org/apache/fineract/integrationtests/DatatableIntegrationTest.java

```diff
@@ -247,6 +252,15 @@ public void validateCreateReadDeleteDatatable() {
         assertEquals(datatableEntryMap.get("itsAString"), ((Map) updatedDatatableEntryResponse.get("changes")).get("itsAString"));
         assertEquals(datatableEntryMap.get("itsAText"), ((Map) updatedDatatableEntryResponse.get("changes")).get("itsAText"));
 
+        FineractClient fineract = FineractClient.builder().insecure(true).baseURL("https://localhost:8443/fineract-provider/api/v1/")
```

Comment:
This can be replaced with the "IntegrationTest.fineract()".

Reply:
replaced